overlay/hooks/entrypoint-pre.d/10_setup.sh

```diff
@@ -4,9 +4,16 @@ set -e
 # Preprocess UPSTREAM_DNS to allow for multiple resolvers using the same syntax as lancache-dns
 UPSTREAM_DNS="$(echo -n "${UPSTREAM_DNS}" | sed 's/[;]/ /g')"
 
+if [ ! -z ${CACHE_MEM_SIZE} ]; then
```

Comment:
As per monolithic this should be done in a deprecation file so all the checks happen in the same place (I'd perhaps suggest 00_generic_deprecation.sh to avoid naming clashes)

https://github.com/lancachenet/monolithic/blob/master/overlay/hooks/entrypoint-pre.d/00_deprecation.sh

Reply:
At the moment it can probably just warning rather than error, then EXPORT CACHE_INDEX_SIZE up to the controlling shell?

Reply:
I'm not sure you can export a variable from one script to the parent shell - unless this is a magic bit of supervisord functionality?
Could move the warning out to a separate script, and leave the action in place?

Reply:
Kinda forgot about this!

I think the best option with fresh eyes is to do the transfer here so it can be used in the script, and then put the warning message into deprecation.sh but without the abort so it will stilll continue for the moment.

We would also need to update the website and docker-compose (I don't think we have PR's waiting for those do we?